Blockzilla/TelemetryIntegration.swift

```diff
@@ -45,9 +45,10 @@ class TelemetryEventObject {
     public static let setting = "setting"
     public static let menu = "menu"
     public static let pasteAndGo = "paste_and_go"
-    public static let copyImage = "copy_image_buuton"
+    public static let copyImage = "copy_image_button"
```

Comment:
What's the change here?

Reply:
Fixes a typo. But these events aren't being used anymore anyway so we could probably take them out in another patch